src/lava/magma/core/model/py/model.py

```diff
@@ -41,7 +41,7 @@ def __init__(self):
         self.py_ports: ty.List[AbstractPyPort] = []
         self.var_ports: ty.List[PyVarPort] = []
         self.var_id_to_var_map: ty.Dict[int, ty.Any] = {}
-        self.proc_params: ty.Dict[str, ty.Any] = {}
+        self.proc_params: ty.Dict[str, ty.Any] = proc_params
```

Comment:
This should probably get pushed up all the way to AbstractProcessModel.

An NcProcModel will need that information as well.

Reply:
Pushed this behavior to AbstractProcessModel